modules/data.land/R/extract_FIA.R

```diff
@@ -114,9 +115,48 @@ extract_FIA <- function(lon, lat, start_date, gridres = 0.075, dbparms){
     logger.debug(paste0(nrow(css.info), " trees remain after removing entries with no dbh, spcd, and/or n."))
   }
     
-  fia.info[[2]] <- css.info
+  #veg_info[[2]] <- css.info
+  obs <- css.info
+  
+  #--------------------------------------------------------------------------------------------------#
+  # Match species : this step requires DB connections 
+  
+  bety <- dplyr::src_postgres(dbname   = dbparms$bety$dbname, 
+                              host     = dbparms$bety$host, 
+                              user     = dbparms$bety$user, 
+                              password = dbparms$bety$password)
+  con  <- bety$con
+  
+  format.name <- "fia" 
+  code.col    <-  "spcd"
+  
+  # match code to species ID
+  spp.info <- match_species_id(input_codes = obs[[code.col]], format_name = format.name, bety = bety)
```

Comment:
yes, this is actually what it is, as I mention in the comment below the first module kind of needs to have two versions load_veg and extract_veg..similar to met.process -> download.raw.met : i) site ii) regional.. only extract_FIA is a special case as it's the only DB source for now, I can refactor this